build-package.sh

```diff
@@ -232,11 +265,13 @@ fi
 # Generate test code coverage report (macOS). The Travis build must have the
 # CODECOV_ELIGIBLE environment variable defined.
 if [ "$(uname)" == "Darwin" -a -n "${CODECOV_ELIGIBLE}" ]; then
+  travis_start "swift_codecov"
```

Comment:
Can we also add `travis_start "swift_sonarcloud"` below on line 281?

Reply:
Done